src/test/java/org/dataloader/DataLoaderTest.java

```diff
@@ -1066,54 +1144,120 @@ private static CacheKey<JsonObject> getJsonObjectCacheMapFn() {
                 .collect(Collectors.joining());
     }
 
-    private static <K, V> DataLoader<K, V> idLoader(DataLoaderOptions options, List<Collection<K>> loadCalls) {
-        return newDataLoader(keys -> {
-            loadCalls.add(new ArrayList<>(keys));
-            @SuppressWarnings("unchecked")
-            List<V> values = keys.stream()
-                    .map(k -> (V) k)
-                    .collect(Collectors.toList());
-            return CompletableFuture.completedFuture(values);
-        }, options);
+    private static Stream<Arguments> dataLoaderFactories() {
+        return Stream.of(
+                Arguments.of(Named.of("List DataLoader", new ListDataLoaderFactory())),
+                Arguments.of(Named.of("Mapped DataLoader", new MappedDataLoaderFactory()))
```

Comment:
Now, if we wish to test more implementations of a `DataLoader`, we need only drop in a new entry here, and we get instant coverage.